lib/byron/test/unit/Cardano/Wallet/Byron/CompatibilitySpec.hs

```diff
@@ -40,3 +53,36 @@ spec = do
             ] $ \bytes ->
             let str = show $ toGenTx $ SealedTx $ unsafeFromHex bytes
             in it (take 23 str) (putStrLn str)
+
+    describe "Conversions" $
+        it "toPoint' . fromTip' == getTipPoint" $ property $ \gh tip -> do
+            let fromTip' = fromTip gh epochLength
+            let toPoint' = toPoint gh epochLength
+            toPoint' (fromTip' tip) === (getTipPoint tip)
+
+instance Arbitrary (Hash "Genesis") where
+    arbitrary = Hash . BS.pack <$> vector 32
+
+instance Arbitrary (Hash "BlockHeader") where
+    arbitrary = Hash . BS.pack <$> vector 32
+
+instance (Arbitrary (Tip ByronBlock)) where
+    arbitrary = oneof
+        [ return TipGenesis
+        , arbitraryTip
+        ]
```

Comment:
Would be slightly better to use `frequency` here and give a lesser weight to `TipGenesis`, otherwise, half of the tests are actually testing the same thing.